bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/services/formsportal/OperationManager.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.adobe.cq.forms.core.components.models.services.formsportal;
+
+import java.util.List;
+
+import com.adobe.cq.forms.core.components.models.formsportal.DraftsAndSubmissions;
+import com.adobe.cq.forms.core.components.models.formsportal.PortalLister;
+
+public interface OperationManager {
```

Comment:
I don't think anyone can customize this interface, please add the related annotation here

Reply:
Added provider type